src/Razor/tools/RazorSyntaxGenerator/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,40 @@
+#nullable enable
```

Comment:
This isn't a shipping product. I recall Doug recently planned to address this

Reply:
Project isn't even packable. See https://github.com/dotnet/aspnetcore/blob/release/5.0-rc1/src/Razor/tools/RazorSyntaxGenerator/RazorSyntaxGenerator.csproj#L10

Reply:
So, why is the analyzer running on it?  Should I just remove these ones?

Reply:
Because we have implementation projects that create assemblies shipped in other packages e.g. anything in the sharedFx without it's own package, the current conditions don't check `$(IsPackable)`. We could complicate the `Condition` further but the simple thing to do right now is remove the `PublicAPI.*.txt` files.

Reply:
Makes sense.  Removed